src/routes/balances/balances.controller.spec.ts

```diff
@@ -221,6 +221,84 @@ describe('Balances Controller (Unit)', () => {
         });
     });
 
+    it('returns large numbers as is (not in scientific notation)', async () => {
+      const chainId = '1';
+      const safeAddress = '0x0000000000000000000000000000000000000001';
+      const transactionApiBalancesResponse = [
+        balanceBuilder()
+          .with('balance', '266279793958307969327868')
+          // The Transaction Service can return scientific notation for large numbers
+          .with('fiatBalance', '6.25164198388829e+43')
+          .with('fiatConversion', '2.347771827128244e+38')
```

Comment:
Would be interesting to know which options faker provides here. We can generate numbers with a minimum value (potentially also with scientific notation).

Reply:
As agree, let's tackle this after the release of the hotifx.